metaworld/benchmarks/ml10.py

```diff
@@ -6,15 +6,30 @@
 
 class ML10(MultiClassMultiTaskEnv, Benchmark):
 
-    def __init__(self, env_type='train', sample_all=False):
+    def __init__(self, env_type='train', sample_all=False, task_name=None):
         assert env_type == 'train' or env_type == 'test'
 
-        cls_dict = MEDIUM_MODE_CLS_DICT[env_type]
-        args_kwargs = MEDIUM_MODE_ARGS_KWARGS[env_type]
+        if task_name is not None:
+            assert task_name in MEDIUM_MODE_CLS_DICT[env_type]
+            "{} does not exist in ML10 {} tasks".format(task_name, env_type)
```

Comment:
I don't think splitting the assert statement like this works. Just use an `if`, `raise ValueError` instead.

Reply:
Fixed